ffmpeg/probe.go

```diff
@@ -47,8 +47,14 @@ type ProbeStream struct {
 	Channels           int              `json:"channels,omitempty"`
 	ChannelLayout      string           `json:"channel_layout,omitempty"`
 	BitsPerSample      int              `json:"bits_per_sample,omitempty"`
+	Tags               StreamTags       `json:"tags,omitempty"`
 }
 
+type StreamTags struct {
```

Comment:
Probably something like:
```
type StreamTags struct {
	Language string `json:"language,omitempty"`
	Duration  string `json:"duration,omitempty"`
}
```

Can you please provide an example of such probe to make sure?